simulation_parameters/common/screen_effects.json

```diff
@@ -0,0 +1,22 @@
+﻿{
+  "none": {
+    "name": "SCREEN_EFFECT_NONE",
+    "index": 0,
+    "ShaderPath": null
+  },
+  "greyscale": {
+    "name": "SCREEN_EFFECT_GREYSCALE",
+    "index": 1,
+    "ShaderPath": "res://shaders/screen_effects/Greyscale.shader"
+  },
+  "gameboy": {
+    "name": "SCREEN_EFFECT_GB",
```

Comment:
I think the translation key should still have the full name. The filename is after all still the full name. I think it's fine to use here, and if the full name is not used here then a translation hint will need to be added on weblate.

Reply:
Well if it is used, a hint not to use the full name will probably also need to be added

Reply:
I think that's a smaller problem than me having to intervene when translators are clueless and end up translating GB to Great Britain equivalent or something.

Reply:
I'll do this change and perform final testing unless you really object.